AUTHORS.rst

```diff
@@ -0,0 +1,5 @@
+============
+Contributors
+============
+
+* Steven Murray <steven.g.murray@asu.edu>
```

Comment:
We should probably add Zac to this list too 😉

Reply:
Yup, done.